packages/headless/src/database/mongodb/product/index.ts

```diff
@@ -5,19 +5,17 @@ import { Injectable } from '@nestjs/common';
 
 @Injectable()
 export class ProductDatabase implements IProductDatabase {
-    constructor() { }
+  async findById(productId: string) {
+    const product = await ProductModel.findOne({ id: productId });
+    return Promise.resolve(product);
```

Comment:
Please keep single line here. I've added similar in my example code. but this seems un-necessary. This should be removed from my example too